### PR TITLE
Update apps.json

### PR DESCRIPTION
**Pull request by novia713**
my addon for Monchi game (https://marketplace.firefox.com/app/monchi/)

### CODE REVIEW

File: app/apps.json [hunk 1]
```diff
@@ -325,5 +325,15 @@
       "type": "addon",
       "url": "https://github.com/justindarc/fxos-animated-clock",
       "revision": "955cf91115a1a7dc4baf1ae6d6f9166bf3184322"
+    },
+    "https://justindarc.github.io/fxos-animated-clock/manifest.webapp": {
```
Comment (mikehenrty) [inline app/apps.json]: This key needs to be the same as your manifestURL below.

File: app/apps.json [hunk 2]
```diff
@@ -325,5 +325,15 @@
...
+      "manifestURL": "https://raw.githubusercontent.com/novia713/monchi-dificil/master/manifest.webapp",
```
Comment (mikehenrty) [inline app/apps.json]: Better is to host this on gh-pages. See here for information on how to do that:

https://help.github.com/articles/creating-project-pages-manually/

File: app/apps.json [hunk 3]
```diff
@@ -325,5 +325,15 @@
...
+      "name": "Difficult mode for Monchi game",
+      "icon": "https://github.com/novia713/monchi-dificil/blob/master/icons/orange-monchi128.png",
```
Comment (mikehenrty) [inline app/apps.json]: This also needs to be a raw image.

**Comment by mikehenrty** [comment]
Hi @novia713 !! thanks for your add-on. Please address my comments :)

**Comment by novia713** [comment]
changes done!
please check them now :)

File: app/apps.json [hunk 4]
```diff
@@ -325,5 +325,15 @@
       "type": "addon",
       "url": "https://github.com/justindarc/fxos-animated-clock",
       "revision": "955cf91115a1a7dc4baf1ae6d6f9166bf3184322"
+    },
+    "http://novia713.github.io/monchi-dificil/manifest.webapp": {
```
Comment (mikehenrty) [inline app/apps.json]: This needs to be https.

File: app/apps.json [hunk 5]
```diff
@@ -325,5 +325,15 @@
...
+      "manifestURL": "http://novia713.github.io/monchi-dificil/manifest.webapp",
```
Comment (mikehenrty) [inline app/apps.json]: https :)

**Comment by mikehenrty** [comment]
Hi @novia713, I'm so sorry to do this to you. Our add-on format changed a few days ago. Here is a guide for converting your add-on from the old format to the new format:
https://etherpad.mozilla.org/converting-fxos-addons

Also, here is a mailing list post about how to submit your new add-on back to the hackerplace:
https://groups.google.com/d/msg/mozilla.dev.b2g/m10n9RpNwlI/utQ4-8JgEQAJ

Let me know if you have any questions. You can find me on the Mozilla IRC server as mhenretty :)

**Comment by mikehenrty** [comment]
Please submit your add-on to the Marketplace, since we are deprecating the Hackerplace.

**Pull request closed without merging**